bolt/client.go

```diff
@@ -47,13 +54,21 @@ func NewClient() *Client {
 
 // Open and initialize boltDB. Initial buckets are created if they do not exist.
 func (c *Client) Open(ctx context.Context) error {
+	if _, err := os.Stat(c.Path); os.IsNotExist(err) {
+		c.isNew = true
+	}
+
 	// Open database file.
 	db, err := bolt.Open(c.Path, 0600, &bolt.Options{Timeout: 1 * time.Second})
 	if err != nil {
 		return err
 	}
 	c.db = db
 
+	return nil
+}
+
+func (c *Client) Initialize(ctx context.Context) error {
```

Comment:
Is it possible to call initialize and update from Open?  it would cut down the code to use bolt (thinking about the rest of the tests in the PR that `Initialize`